src/main/java/com/google/neighborgood/servlets/UserInfoServlet.java

```diff
@@ -94,6 +148,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       entity = new Entity("UserInfo");
       entity.setProperty("nickname", nickname);
       entity.setProperty("address", address);
+      entity.setProperty("zipcode", "59715");
+      entity.setProperty("country", "United States");
```

Comment:
I made some changes to the doGet() function in the branch leonard-map. Maybe it will cause a confilct with your file.

Reply:
I'll check the branch out to see what conficts we can avoid :) 
thanks for the heads up!

Reply:
Ah I just saw what you meant, I'll keep an eye out when merging the branch but will leave those hardcoded for now since we can't test my branch with the real zipcode, country values yet :)